packages/kununu-utils/hooks/useTrapFocus/__tests__/useTrapFocus.spec.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {
+  render, cleanup, fireEvent,
+} from '@testing-library/react';
+
+import useTrapFocus from '..';
+
+function TrapFocusComponent () {
+  const ref = useTrapFocus();
+
+  return (
+    <div ref={ref}>
+      <button type="button">First Button</button>
+      <p>Text</p>
+      <button type="button">Middle Button</button>
+      <p>Text</p>
+      <button type="button">Last Button</button>
+    </div>
+  );
+}
+
+describe('useTrapFocus', () => {
+  afterEach(cleanup);
```

Comment:
I thought cleanup was unnecessary when using jest 🤔 because of [this](https://testing-library.com/docs/react-testing-library/api/#cleanup).

Reply:
that's super cool, didn't know about that, will update right away :the_horns:

Reply:
I don't knew about it as well. Pretty sure me and @samuelsilvadev aren't the only ones. 
@frodrigues-kununu  Please shared this on the #frontend